examples/step-68/doc/results.dox

```diff
@@ -0,0 +1,90 @@
+<h1>Results</h1>
+
+The directory in which this program is run contains an example parameter file by default.
+If you do not specify a parameter file as an argument on the command
+line, the program will try to read the file "parameters.prm" by default, and
+will execute the two dimensional version of the code.
+
+On any number of core, the simulation output will look like:
+
+@code
+bash$ mpirun -np 4 ./step-68 parameters.prm
+Number of particles inserted: 606
+Repartitioning triangulation after particle generation
+Writing particle output file: analytical-particles-0
+Writing background field file: background-0
+Writing particle output file: analytical-particles-10
+Writing background field file: background-10
+Writing particle output file: analytical-particles-20
+Writing background field file: background-20
+Writing particle output file: analytical-particles-30
+Writing background field file: background-30
+...
+Number of particles inserted: 606
+Repartitioning triangulation after particle generation
+Writing particle output file: analytical-particles-0
+Writing background field file: background-0
+Writing particle output file: analytical-particles-10
+Writing background field file: background-10
+Writing particle output file: analytical-particles-20
+Writing background field file: background-20
+Writing particle output file: analytical-particles-30
+Writing background field file: background-30
+...
+Writing particle output file: interpolated-particles-1980
+Writing background field file: background-1980
+Writing particle output file: interpolated-particles-1990
+Writing background field file: background-1990
+Writing particle output file: interpolated-particles-2000
+Writing background field file: background-2000
+@endcode
+
+We note that, by default, the simulation runs the particle tracking with
+an analytical velocity for 2000 iterations, then runs the particle tracking with
+velocity interpolationn for the same duration. The results are written every
+10th iteration.
+
+<h3> Motion of the particles </h3>
+
+The following animation displays the trajectory of the particles as they
+are advected by the flow field. We see that after the complete duration of the
+flow, the particle go back to their initial configuration as is expected.
+
+@htmlonly
+<p align="center">
+  <iframe width="560" height="500" src="https://youtu.be/EbgS5Ch35Xs"
+   frameborder="0"
+   allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+   allowfullscreen></iframe>
+ </p>
+@endhtmlonly
+
+<h3> Dynamic load balancing </h3>
+
+The following animation shows the impact of dynamic load balancing. We clearly
```

Comment:
This section comes as somewhat of a surprise, because load balancing is not discussed in the introduction or elsewhere outside of the code. Do you want me to write a paragraph on why it is important when handling particles in distributed triangulations?